spec.md

```diff
@@ -1374,7 +1374,7 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. |
+| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. If credentials have been stored in a format that does not allow retrieval of a usable result (e.g. a salted and hashed password), the broker MAY omit this field. This can result in a return value that is not usable to access the service. |
```

Comment:
I would suggest we just remove that sentence.

re: async-binding... perhaps we need to add something else here to make it clear that each GET can decide when to return the credentials.  Meaning, the first GET might do so, but then subsequent GETs may not because they're stored in a hash. This allows for a one-time-only retrieval so async bindings work, but after that things remain secure.  But, in the case of async it MUST support at least one GET of the creds.

Reply:
Is removing that sentence acceptable?

Reply:
`If credentials have been stored in a format that does not allow retrieval of a usable result the broker MAY omit this field.`
@n3wscott @mattmcneeney Is only that sentence being added acceptable? I feel like it makes things murkier rather than clearer without the example.